CARET_trace/test/test_trace_node.cpp

```diff
@@ -135,24 +142,36 @@ TEST(CaretTraceTest, TestStartTransition)
 
 TEST_F(CaretTraceNodeTest, TestWaitTransition)
 {
+  // set_lttng_session_running_return_value(true);
   init_context();
 
-  {  // subscribe start message: WAIT -> PREPARE
+  {  // subscribe start message without lttng session: WAIT -> WAIT
     set_status(TRACE_STATUS::WAIT);
 
     EXPECT_TRUE(node_->get_status() == TRACE_STATUS::WAIT);
-
+    set_lttng_session_running_return_value(false);
     auto start_msg = get_start_msg();
     node_->start_callback(std::move(start_msg));
 
-    EXPECT_TRUE(node_->get_status() == TRACE_STATUS::PREPARE);
+    EXPECT_TRUE(node_->get_status() == TRACE_STATUS::WAIT);
   }
 
-  {  // subscribe end message: WAIT -> WAIT
+  {  // subscribe start message with lttng session: WAIT -> PREPARE
     set_status(TRACE_STATUS::WAIT);
 
     EXPECT_TRUE(node_->get_status() == TRACE_STATUS::WAIT);
+    set_lttng_session_running_return_value(true);
+    auto start_msg = get_start_msg();
+    node_->start_callback(std::move(start_msg));
+
+    EXPECT_TRUE(node_->get_status() == TRACE_STATUS::PREPARE);
+  }
 
+  {  // subscribe end message without lttng session: WAIT -> WAIT
+    set_lttng_session_running_return_value(false);
```

Comment:
I think this line can be removed.

Reply:
Fixed in [f53bfac](https://github.com/tier4/caret_trace/pull/262/commits/f53bfac8e8e37f47b05e2e43822018e8433cd85e).
This commit reflects the all comments below.